health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1286,36 +1461,45 @@
 }
 
 async function getCodesTarget(request: any, localizationMap?: any) {
+    let boundaryCodesWhoseTargetsHasToBeUpdated: any = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'boundaryCodesWhoseTargetsHasToBeUpdated' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)